js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -77,7 +80,29 @@ describe("Attestation", () => {
           employees={EMPLOYEES}
         />,
       );
-      expect(view.getByRole("textbox")).toHaveValue("123");
+      const input = view.getByLabelText(/Operator Badge Number/i, {
+        selector: "input",
+      });
+      expect(input).toHaveValue("123");
+    });
+  });
+
+  describe("radio text box", () => {
+    test("it's there", () => {
```

Comment:
The presence of the input is already tested by the other tests that also test filling it out. So I don't think this test adds anything and it could be removed.